src/ResourceTile/ResourceTile.less

```diff
@@ -0,0 +1,186 @@
+@import (reference) "../less/index";
+
+// The Dewey sizing mixins aren't granular enough, so we're using custom values
+@iconSizeSmall: 5rem;
+@iconSizeMedium: 6.5rem;
+@iconSizeLarge: 8rem;
+@resourceTilePaddingSmall: 0.75rem;
+@resourceTilePaddingMedium: 0.875rem;
+@resourceTilePaddingLarge: 1rem;
+@fontSizeSmall: 0.75rem;
+@fontSizeMedium: 0.875rem;
+@fontSizeLarge: 1rem;
+
+.ResourceTile {
```

Comment:
Ooh one more thought: For components that have names that could actually conflict with in-repo components, we sometimes prefix the component styles with `dewey--`. We do this in the Dewey `TopBar` for example.

https://github.com/Clever/components/blob/e68b5faf67fa7e756e95445a0493e291ffb26a98/src/TopBar/index.less#L4

Might make sense to do that for this component as well. I wouldn't be surprised if someone set up a component named `<ResourceTile>` and had trouble styling it because Dewey styles were getting applied to it, not realizing that that was happening.